src/content/en/2020/markup.md

```diff
@@ -0,0 +1,626 @@
+
+<div class="note">
+
+#### `isindex`
+
+If you were wondering: The [`isindex` element](https://www.w3.org/TR/html401/interact/forms.html#edef-ISINDEX) was present once. (It was part of the HTML specs [until version 4.01, and of XHTML in 1.0](https://meiert.com/en/indices/html-elements/), yet only properly [specified in 2006](https://lists.w3.org/Archives/Public/public-whatwg-archive/2006Feb/0111.html), and [removed in 2016](https://github.com/whatwg/html/pull/1095).)
+
+</div>
+
+### Proprietary and made-up elements
+
+In our set of elements we found some that were neither standard HTML (nor SVG nor MathML) elements, nor custom ones, nor obsolete ones, but somewhat proprietary ones. The Top 10 that we identified were the following:
+
+<figure markdown>
+| Element | Percentage of pages with this element |
+|---|---|
+| `noindex` | 0.89% |
+| `jdiv` | 0.85% |
+| `mediaelementwrapper` | 0.49% |
+| `ymaps` | 0.26% |
+| `yatag` | 0.20% |
+| `ss` | 0.11% |
+| `include` | 0.08% |
+| `olark` | 0.07% |
+| `h7` | 0.06% |
+| `limespot` | 0.05% |
+<figcaption>{{ figure_link(caption="Elements of questionable heritage.", sheets_gid="184700688", sql_file="pages_element_count_by_device_and_element_type_present.sql") }}</figcaption>
+</figure>
+
+The source of these elements appears to be mixed, as in some are unknown while others can be traced. The most popular one, `noindex`, is probably due to [Yandex’s recommendation](https://yandex.com/support/webmaster/adding-site/indexing-prohibition.html) of it to prohibit page indexing. `jdiv` was noted in last year’s Almanac and is from JivoChat. `mediaelementwrapper` comes from the MediaElement media player. Both `ymaps` and `yatag` are also from Yandex. The `ss` element could be from ProStores, a former ecommerce product from eBay, and `olark` may be from the Olark chat software. `h7` appears to be a mistake. `limespot` is probably related to the Limespot personalization program for ecommerce. None of these elements are part of a web standard.
+
+### Headings
+
+[Headings](https://html.spec.whatwg.org/multipage/dom.html#heading-content) make for a special category of elements that play an important role in [sectioning](https://html.spec.whatwg.org/multipage/dom.html#sectioning-content-2) and for [accessibility](https://www.w3.org/WAI/tutorials/page-structure/headings/).
+
+<figure markdown>
+| Heading | Occurrences | Average per page |
+|---|---|---|
+| `h1` | 10,524,810 | 1.66 |
+| `h2` | 37,312,338 | 5.88 |
+| `h3` | 44,135,313 | 6.96 |
+| `h4` | 20,473,598 | 3.23 |
+| `h5` | 8,594,500 | 1.36 |
+| `h6` | 3,527,470 | 0.56 |
+<figcaption>{{ figure_link(caption="Frequency and average use of standard heading elements,", sheets_gid="277662548", sql_file="pages_wpt_bodies_by_device_and_percentile_and_heading_level.sql") }}</figcaption>
```

Comment:
Change comma to fullstop:

```suggestion
<figcaption>{{ figure_link(caption="Frequency and average use of standard heading elements.", sheets_gid="277662548", sql_file="pages_wpt_bodies_by_device_and_percentile_and_heading_level.sql") }}</figcaption>
```

Few instances of this and not always easy to spot so pointing out now.

Reply:
Interesting. Will check the whole doc for these.